tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-confusion-matrix.html

```diff
@@ -0,0 +1,256 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<link rel="import" href="../polymer/polymer.html">
+
+<!-- Displays a simple confusion matrix. -->
+<dom-module id="tf-confusion-matrix">
+  <template>
+    <style>
+      table {
+        display: inline-block;
+      }
+      td {
+        text-align: right;
+      }
+      th, td {
+        padding: 3px;
+        font-weight: normal;
+      }
+      .flex {
+        display: flex;
+        flex-direction: row-reverse;
+      }
+      .pred-label {
+        text-align: left;
+        font-size: 10px;
+      }
+      .actual-label {
+        text-align: right;
+        font-size: 10px;
+      }
+      .n-label {
+        font-size: 10px;
+        vertical-align: bottom;
+      }
+      .correct {
+        border-left: 3px solid DarkSeaGreen;
+      }
+      .incorrect {
+        border-left: 3px solid lightcoral;
+      }
+      .blank {
+        background: darkgray;
+      }
+      .parenthetical {
+        padding-right: 5px;
+        color: gray;
+        width: 50px;
+      }
+    </style>
+    <div id="holder"></div>
+  </template>
+
+  <script>
+    "use strict";
+
+    Polymer({
+      is: "tf-confusion-matrix",
+      properties: {
+        counts: {type: Object, observer: 'drawMatrix'},
+      },
+
+      ready: function() {
+        // Allows d3-created elements in this Polymer element to be styled by
+        // the css rules above.
+        this.scopeSubtree(this.$.holder, true);
+      },
+
+      drawMatrix: function(counts) {
+        this.$.holder.innerHTML = '';
+        if (!counts) {
+          return;
+        }
+
+        // Get all labels from the counts dict.
+        let allLabels = {};
+        const outerLabelList = Object.keys(counts);
+        for (let i = 0; i < outerLabelList.length; i++) {
+          allLabels[outerLabelList[i]] = true;
+          const innerLabelList = Object.keys(counts[outerLabelList[i]]);
+          for (let j = 0; j < innerLabelList.length; j++) {
+            allLabels[innerLabelList[j]] = true;
+          }
+        }
+        let labelsList = Object.keys(allLabels).sort();
+
+        // For binary classification confusion matrices, we want the 'Yes'
+        // column/row to appear before 'No', so we reverse this labels list.
+        if (labelsList.length == 2 && labelsList[0] == 'No'
+            && labelsList[1] == 'Yes') {
+          labelsList = labelsList.reverse();
+        }
+
+        const columnTotals = new Array(labelsList.length);
+
+        // Create the confusion matrix from the counts dict.
+        const countMatrix = [];
+        for (let i = 0; i < labelsList.length; i++) {
+          columnTotals[i] = 0;
+          countMatrix.push(new Array(labelsList.length).fill(0));
```

Comment:
TB does not officially support IE11 but `fill` is not supported there.
https://lodash.com/docs/4.17.10#fill might be a good alternative without a Polyfill.